XML/TestCases/FunctionalTests-TVM.xml

```diff
@@ -0,0 +1,87 @@
+<TestCases>
+  <test>
+    <testName>VALIDATE-SECUREBOOT-COMPATIBILITY-FOR-UBUNTU18_04</testName>
+    <testScript>TVM-TestSecureBootCompatibility.sh</testScript>
+    <files>.\Testscripts\Linux\TVM-TestSecureBootCompatibility.sh,.\Testscripts\Linux\utils.sh,.\Tools\sbinfo</files>
+    <Platform>Azure</Platform>
+    <Category>Functional</Category>
+    <Area>TVM</Area>
+    <Tags>SecureBoot</Tags>
+    <Priority></Priority>
+    <SetupConfig>
+      <TestLocation>eastus2euap</TestLocation>
+      <SetupType>OneVM</SetupType>
+      <OverrideVMSize>Standard_D2s_v3</OverrideVMSize>
+      <VMGeneration>2</VMGeneration>
+      <ARMImageName>Canonical 0003-com-ubuntu-server-trusted-vm 18_04-gen2 18.04.202004290</ARMImageName>
```

Comment:
@harz566 we can put multiple image in one case?

Reply:
yes, it can be placed with comma-separated value

Reply:
I tried it, but it didn't work as expected.

Reply:
@harz566 Hmmm it sounds a new bug on this feature. Can you take a look at it separately?

Reply:
This is not the way you can create / run the test. Please **keep one test case**. When you run the test, please use "ARMimagename" parameter taking the different distro names.

Reply:
I updated the test cases, please take a look.